staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -955,3 +959,10 @@ type MemoryReservation struct {
 	NumaNode int32           `json:"numaNode"`
 	Limits   v1.ResourceList `json:"limits"`
 }
+
+// Configure swap memory available to container workloads. May be one of
+// "", "LimitedSwap": workload combined memory and swap usage cannot exceed pod memory limit
+// "UnlimitedSwap": workloads can use unlimited swap, up to the allocatable limit.
```

Comment:
move the comments about allowed values to be field doc on the `SwapBehavior` field (and start with `swapBehavior` to match the serialized field name (and the update being made in https://github.com/kubernetes/kubernetes/pull/96699)